lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -841,12 +836,33 @@ class TaskSyntheticFrontEnd : public SyntheticChildrenFrontEnd {
             raw_pointer_type);
       }
       return m_address_sp;
-    case 1:
+    case 1: {
+      // TypeMangling for "Swift.UInt64"
+      CompilerType uint64_type =
+          m_ts->GetTypeFromMangledTypename(ConstString("$ss6UInt64VD"));
```

Comment:
there is `TypeSystem::GetPointersizedIntType()`, but generally having a helper to get a uint64 in typesystemswift would make sense.

Reply:
It appears this is only in `TypeSystemClang`, but seems reasonable to add to Swift separately.